docs/conversion_examples_gallery/behavior/lightningpose.rst

```diff
@@ -23,7 +23,7 @@ Convert LightningPose pose estimation data to NWB using :py:class:`~neuroconv.da
     >>> labeled_video_file_path = str(folder_path / "labeled_videos/test_vid_labeled.mp4")
 
     >>> converter = LightningPoseConverter(file_path=file_path, original_video_file_path=original_video_file_path, labeled_video_file_path=labeled_video_file_path, verbose=False)
-
+    Source data is valid!
```

Comment:
Here I was thinking you despised verbosity in the testing suite 😉

Reply:
I do, but I did not want to change there because I saw that @pauladkisson explicitly set `verbose=True` so I went with what I assumed was the intent.